docs/docs/guides/sveltekit.mdx

```diff
@@ -0,0 +1,162 @@
+---
+title: Sveltekit + Hanko
+sidebar_label: Sveltekit
+keywords: [svelte, sveltekit]
+sidebar_custom_props:
+  docCardIconName: svelte
+---
+
+
+# Integrate Hanko with Sveltekit
+
+Learn how to quickly add authentication and user profile in your Sveltekit app using Hanko.
+
+## Install `@teamhanko/hanko-elements`
+
+Once you've initialized your app, installing hanko-elements provides you with access to the prebuilt components: `hanko-auth` and `hanko-profile`.
+
+```shell npm2yarn
+npm install @teamhanko/hanko-elements
+```
+
+## Add the Hanko API URL
+
+Retrieve the API URL from the [Hanko console](https://cloud.hanko.io/) and place it in your `.env` file.
+
+```sh title=".env"
+PUBLIC_HANKO_API_URL=https://f4****-4802-49ad-8e0b-3d3****ab32.hanko.io
+```
+
+:::info
+  If you are self-hosting you need to provide the URL of your running Hanko
+  backend.
+:::
+
+## Add `<hanko-auth>` component
+
+The `<hanko-auth>` web component adds a login interface to your app. Begin by importing the register function from `@teamhanko/hanko-elements` into your Svelte component. Call it with the Hanko API URL as an argument to register `<hanko-auth>` with the browser's [CustomElementRegistry](https://developer.mozilla.org/de/docs/Web/API/CustomElementRegistry). Once done, use the `<hanko-auth>` element in your component.
+
+Here, we utilize the `onAuthflowCompleted` custom event, which is triggered upon the successful completion of the auth flow, redirecting the user to a designated route.
+
+```js title="HankoAuth.svelte" showLineNumbers
+<script>
+  import { onMount } from "svelte";
+  import { goto } from "$app/navigation";
+  import { register } from "@teamhanko/hanko-elements";
+  import { env } from "$env/dynamic/public";
+  const hankoApi = env.PUBLIC_HANKO_API_URL;
+
+  const redirectAfterLogin = () => {
+    goto("/dashboard");
+  };
+
+  onMount(async () => {
+    register(hankoApi).catch((error) => {
+      // handle error
+    });
+  });
+</script>
+
+<hanko-auth on:onAuthFlowCompleted={redirectAfterLogin} />
+```
+
+:::info
+When incorporating any Hanko component in your `+page.svelte` file(routes), it is necessary to disable server-side rendering (SSR) by setting `ssr` to false in your `+layout.ts` file.
```

Comment:
We can add it to `+layout.ts` as well, as it was not a root layout. I added that layout for that specific page. Yeah, but you're right, if we have multiple pages coming from the same route, this will apply to all the routes, which the user might not want.